src/main/webapp/WEB-INF/tags/instructor/studentList/filterBox.tag

```diff
@@ -0,0 +1,75 @@
+<%@ tag description="instructorStudentList - Student filter box" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ attribute name="filterboxdata" type="teammates.ui.template.InstructorStudentListFilterBox" required="true" %>
```

Comment:
I have to say I don't understand the purpose behind appending `data` to everyhing...
But either way, we should probably put attributes in camelCase in tags / jsps too.